package.json

```diff
@@ -25,9 +26,11 @@
     "multiline": "^1.0.2",
     "nyc": "^10.3.0",
     "package-json": "^2.4.0",
+    "power-assert": "^1.4.2",
     "propprop": "^0.3.1",
     "semver": "^5.3.0",
     "shelljs": "^0.7.3",
+    "sinon": "^2.2.0",
```

Comment:
Are we using this? (sorry for the repeated comments-- just don't want to forget about this)